stellargraph/mapper/mini_batch_node_generators.py

```diff
@@ -62,7 +63,7 @@ class ClusterNodeGenerator(Generator):
 
     def __init__(self, G, clusters=1, q=1, lam=0.1, name=None):
 
-        if not isinstance(G, StellarGraph):
+        if not isinstance(G, (StellarGraph, Neo4jStellarGraph)):
```

Comment:
We need a `_StellarGraphBase` base class or something to both more clearly define an API and also make these sort of checks easy 😄 

Is there an issue for this? If not, could you open one?

Reply:
ah i thought we had one but turns out we didn't - I filed #1620 😄